src/components/projects/projects-table/projects-table/projects-table.tsx

```diff
@@ -34,11 +34,52 @@ const ProjectsTable: React.FC<DisplayProjectProps> = ({ allProjects }: DisplayPr
   // TODO: Sort by wbsNum means 1.1.0 > 1.12.0 > 1.2.0, but desired is 1.1.0 > 1.2.0 > 1.12.0
   // TODO: Sort by duration means 12 > 2 > 4 > 5 > 9, but desired is 12 > 9 > 5 > 4 > 2
   const columns: ColumnDescription[] = [
-    { dataField: 'wbsNum', text: 'WBS #', align: 'center', sort: true },
-    { dataField: 'name', text: 'Name', align: 'left', sort: true },
-    { dataField: 'projectLead', text: 'Project Lead', align: 'left', sort: true },
-    { dataField: 'projectManager', text: 'Project Manager', align: 'left', sort: true },
-    { dataField: 'duration', text: 'Duration', align: 'center', sort: true }
+    {
+      headerAlign: 'center',
+      dataField: 'wbsNum',
+      text: 'WBS #',
+      align: 'center',
+      sort: true,
+      sortFunc: (a, b, order: SortOrder) => {
+        const a1 = a.split('.');
+        const b1 = b.split('.');
+        const len = Math.min(a1.length, b1.length);
+        for (let i = 0; i < len; i++) {
+          const a2 = +a1[i] || 0;
+          const b2 = +b1[i] || 0;
+          if (a2 !== b2) {
+            if (order === 'asc') {
```

Comment:
here you're comparing a value which is of the `SortOrder` type against a string. I don't recall specifically what the `SortOrder` type is, but is it not possible to compare against `SortOrder.<SOMETHING>` instead of against a hard-coded string? Hard-coded strings are inherently less flexible

Reply:
We've been looking for a SortOrder field for "asc" or "des", but it seems like there aren't any. As it stands, SortOrder can only equal these two strings, so a hard-coded string might be the only option. If you know of any solutions, we would like to know.